release-notes/swan-lake-beta4-release-note.md

```diff
@@ -298,6 +298,32 @@ float f13 = 0x1A.0;
 float f14 = 0x1A.0p4;
 ```
 
+- Disallowed intervening white spaces in the qualified identifier to avoid parsing an ambiguity between the ternary conditional expression and qualified identifier.
```

Comment:
Shouldn't this be
```suggestion
- Disallowed intervening white spaces in the qualified identifier to avoid a parsing ambiguity between the ternary conditional expression and qualified identifier.
```

Not sure what "to avoid parsing an ambiguity" means.

Reply:
Also to maintain consistency, I think this should be

```suggestion
- Intervening white spaces have been disallowed in the qualified identifier to avoid a parsing ambiguity between the ternary conditional expression and qualified identifier.
```